IdentityServer/v7/docs/content/quickstarts/1_client_credentials.md

```diff
@@ -226,14 +234,16 @@ not present.
 {{% /notice %}}
 
 ## Create an API Project
+
 Next, add an API project to your solution. This API will serve protected
 resources that will be secured by IdentityServer.
 
 You can either use the ASP.NET Core Web API template from Visual Studio or use
 the .NET CLI to create the API project. To use the CLI, run the
-following command from the *src* directory:
+following commands:
 
 ```console
+cd src
```

Comment:
This is the first console snippet which (currently) does not contain the `cd` command and relies on the "from the *src* directory" text above. It actually caught me out when following the quickstart because I kept just copying and running the commands as with the other snippets. It seems better to be consistent and help people avoid making that same mistake.